src/Fields/Fields.jl

```diff
@@ -198,6 +207,14 @@ function Base.zeros(::Type{FT}, space::AbstractSpace) where {FT}
 end
 Base.zeros(space::AbstractSpace) = zeros(Spaces.undertype(space), space)
 
+"""
+    ones([FT, ]space::AbstractSpace)
```

Comment:
FT other than the space undertype FT currently doesn't work

Reply:
okay, maybe we should just leave that undocumented.

Reply:
yeah it might be confusing because it differs a bit from Base behavior